cmake/InstallAndPackage.cmake

```diff
@@ -40,8 +40,16 @@ install(FILES
 # A Linux manual only makes sense when using FHS. Otherwise it is a very odd
 # file with little context to what it is.
 if (OPTION_INSTALL_FHS)
+    set(MAN_SOURCE_FILE ${CMAKE_SOURCE_DIR}/docs/openttd.6)
+    set(MAN_BINARY_FILE ${CMAKE_BINARY_DIR}/docs/${BINARY_NAME}.6)
+    install(CODE
+            "
+                execute_process(COMMAND ${CMAKE_COMMAND} -E copy ${MAN_SOURCE_FILE} ${MAN_BINARY_FILE})
```

Comment:
https://stackoverflow.com/a/34800667/995325 ?

Reply:
yeah `configure_file` works here too, anyway the file is always copied during install, because `gzip` compresses it in-place :)

Reply:
Very few of the build systems I've ever worked with (as a packager myself) have automatically compressed the man page. The few times they have, about half the time it's done incorrectly, e.g. not with `gzip -9`, or using `bzip2` as the compressor, or something, and I have to redo it in my script.

For instance, [`wesnoth/wesnoth`](https://github.com/wesnoth/wesnoth) doesn't [compress its man pages upon install](https://github.com/wesnoth/wesnoth/blob/master/doc/man/CMakeLists.txt#L11), and that's just fine with me.